src/Arcus.Security.KeyVault/Factories/KeyVaultClientFactory.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Azure.KeyVault;
+
+namespace Arcus.Security.KeyVault.Factories
+{
+    /// <summary>
+    /// Abstract class to create <see cref="KeyVaultClient"/> object
+    /// </summary>
+    public abstract class KeyVaultClientFactory
```

Comment:
Just curious - Why did you go for an abstract class over an interface?

Reply:
Because I think there might be common methods that can fit there.  I hesitated and went for the abstract class (which is also quite common in Factory patterns - at least in the good old times :))